crates/subspace-node/res/chain-spec-raw-gemini-3a.json

```diff
@@ -18,7 +18,11 @@
   "properties": {
     "ss58Format": 2254,
     "tokenDecimals": 18,
-    "tokenSymbol": "tSSC"
+    "tokenSymbol": "tSSC",
+    "dsnBootstrapNodes": [
+      "/dns/bootstrap-0.gemini-3a.subspace.network/tcp/50000/p2p/12D3KooWMgttYArrdmU5cWZLSShbSoB9teW5dgquqcn5s3kq4PTu",
+      "/dns/bootstrap-1.gemini-3a.subspace.network/tcp/50000/p2p/12D3KooWKSnr5f8XbyaNhCSziYcML9Fem5tYrUj6RzW1yEyMaXKi"
```

Comment:
I don't think these ports are correct. The default is `30433` and are you sure these identities are correct? I would have expected them to be the same as bootnodes above since they use the same network keypair (unless this is a separate bootstrap node application, which I don't necessarily see why we need one here).

Reply:
This is the dsn-bootstrap-node here - https://github.com/subspace/infra/pull/57/files#diff-5590b35d2784444e1e10fc56479fd55a041bc2e072242b5093d29420b273bc48R11

There are two services running
- Regular boostrap nodes
- DSN boostrap nodes with their own keypair

This was result of my discussion with shamil. Is this incorrect ?

Reply:
@shamil-gadelshin do we really need a separate bootstrap node application though if we have node already?

Reply:
I have for now set them to regular bootstrap nodes with updated port.

Reply:
Any peer in the DSN could serve as the bootstrap node, however, bootstrap nodes' PeerID could be defined by its keypair argument and it's convenient to use. Also, we planned to use a set of interconnected DSN bootstrap nodes previously. Answering your question - no, it's not a mandatory requirement.

Reply:
It is already defined on Substrate side by an argument and we can still configure those nodes to be interconnected.